components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -299,6 +407,45 @@ export function CreateWorkspacePage() {
     );
 }
 
+function RememberOptions(params: {
+    disabled?: boolean;
+    checked: boolean;
+    workspaceContext?: WorkspaceContext | null;
+    onChange: (checked: boolean) => void;
+}) {
+    const { disabled, checked, workspaceContext, onChange } = params;
+
+    const label =
+        "Always use these options for " +
+        (CommitContext.is(workspaceContext)
+            ? workspaceContext.repository.owner + "/" + workspaceContext.repository.name
+            : "this repository") +
+        ".";
```

Comment:
**suggestion**: WDYT of 1️⃣ using a muted version of checkbox component to make this more subtle (less annoying), 2️⃣ placing it below the action button, 3️⃣ showing it only when there's a valid repository selected above, and 4️⃣ using a single line for a less intrusive action.

| BEFORE | AFTER |
|-|-|
| <img width="1512" alt="Screenshot 2023-05-02 at 21 19 56" src="https://user-images.githubusercontent.com/120486/235752441-780cc09d-70d0-47d1-a6d1-b8b46a77ecff.png"> | <img width="1512" alt="Screenshot 2023-05-02 at 21 22 00" src="https://user-images.githubusercontent.com/120486/235752455-183593c8-15ca-437c-a881-025a12f3f15d.png"> |